src/components/home-page/hero-section/index.tsx

```diff
@@ -1,50 +1,41 @@
 import Button from 'components/common/button';
 import Image from 'next/image';
 import cn from 'utils';
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+import Headline from './component/Headline';
 
-const height = 'h-[calc(100vh_-_theme(space.20)_-_theme(space.12))]';
+const height = 'h-[calc(100vh_-_theme(space.20))]';
 
-const HeroSection = () => (
+const HeroSection = () => {
+    return (
     <section
         title="hero"
         className={cn(
-            'relative flex w-screen flex-col items-center justify-center bg-black sm:flex-row sm:justify-start',
+            'relative top-20 flex w-full flex-col items-center justify-center bg-black px-4 md:flex-row md:justify-start sm:px-8 md:mt-0 md:px-16 lg:px-32 xl:px-64',
             height,
         )}
     >
-        <div
-            className={cn('absolute -left-3/4 z-0 w-screen', height)}
+        {/* <div
             style={{
                 background:
-                    'radial-gradient(50% 50% at 50% 50%, rgb(106, 108, 253) 0%, rgba(0, 0, 0, 0) 100%)',
+                    'radial-gradient(45.5% 47.88% at 45% 50.47%, #F07546 0%, rgba(0, 0, 0, 0) 100%)',
             }}
-        />
-        <div
-            style={{
-                background:
-                    'radial-gradient(45.5% 47.88% at 45% 50.47%, rgb(255, 0, 199) 0%, rgba(0, 0, 0, 0) 100%)',
-            }}
-            className={`relative top-32 z-0 w-screen sm:absolute sm:top-0 sm:right-0 sm:${height} sm:w-1/2`}
+            className={`hidden md: relative top-32 z-0 hidden w-screen`}
         >
-            <Image src="/hero.svg" alt="hero background" layout="fill" />
-            <img className="invisible" src="/hero.svg" alt="placeholder"></img>
-        </div>
-        <div className="relative -top-[10%] z-40 flex flex-col gap-8 px-5 text-left sm:top-0 xl:ml-32">
-            <h1 className="text-6xl font-medium text-white">
-                Bounties.
+        </div> */}
+        <div className="relative z-40 flex w-full h-full flex-col items-center md:items-start justify-evenly text-left">
+            <h1 className="text-justify text-[38px] font-medium leading-snug sm:leading-normal text-white sm:text-6xl ">
```

Comment:
```suggestion
            <h1 className="text-justify text-[insert-predefined-name-here] font-medium leading-snug sm:leading-normal text-white sm:text-6xl ">
```

We also need to standardize text (`Header1`, `Header2`, `Paragraph`, etc...), either by [extracting these using `@apply`](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply), or by using regular React components.